### PR TITLE
Ajd chapter14

### PR DESCRIPTION
**Pull request by alexdeas**
Chapter 14 answers

### CODE REVIEW

File: ajd/My-List-Util/lib/My/List/Util.pm [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+=cut
+
+sub sum {
+
+  my $class = shift @_;
+  my $sum = 0;
+
+  foreach my $element ( @_ ) {
+    # I could get a module to do this check, but I don't want to
+    if ( $element =~ /^-?\d+(?:\.\d+)?$/ ) {
```
Comment (utopia-2) [inline ajd/My-List-Util/lib/My/List/Util.pm:45]: This won't handle the case where perl stringifies the number using scientific notation - ran into this same thing when adding tests to mine.

File: ajd/My-List-Util/lib/My/List/Util.pm [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+  return $sum;
+
+}
+
+=head2 shuffle
+
+Shuffles the supplied list
+
+Uses a modern Fisher-Yates algorithm
```
Comment (utopia-2) [inline ajd/My-List-Util/lib/My/List/Util.pm:58]: Modern? It's gotta be 60+ years old by now.

Comment (alexdeas) [inline ajd/My-List-Util/lib/My/List/Util.pm:58]: More modern than the older version?

Comment (utopia-2) [inline ajd/My-List-Util/lib/My/List/Util.pm:58]: ... very well. I'll allow this. :cat:

**Pull request merged**